src/dysh/fits/gbtfitsload.py

```diff
@@ -1847,11 +1847,23 @@ def _update_radesys(self):
 
         # Azimuth and elevation case.
         azel_mask = (self["CTYPE2"] == "AZ") & (self["CTYPE3"] == "EL")
+        # Update self._index.
         self._index.loc[azel_mask, "RADESYS"] = radesys["AzEl"]
+        # Update SDFITSLoad.index.
+        sdf_idx = set(self["FITSINDEX"][azel_mask])
+        for i in sdf_idx:
+            sdfi = self._sdf[i].index()
+            azel_mask = (sdfi["CTYPE2"] == "AZ") & (sdfi["CTYPE3"] == "EL")
+            sdfi.loc[azel_mask, "RADESYS"] = radesys["AzEl"]
 
         # Hour angle and declination case.
         hadec_mask = self["CTYPE2"] == "HA"
         self._index.loc[hadec_mask, "RADESYS"] = radesys["HADec"]
+        sdf_idx = set(self["FITSINDEX"][hadec_mask])
+        for i in sdf_idx:
+            sdfi = self._sdf[i].index()
+            azel_mask = sdfi["CTYPE2"] == "HA"
```

Comment:
for consistency with the previous loop, this variable should be hadec_mask.  doesn't change the functionality at all though